docs/api/webviews.md

```diff
@@ -4,127 +4,435 @@ Web views are native components that render content not natively supported by th
 However, elements within web views are not native components, making direct interaction through Detox challenging.
 To address this, Detox offers a suite of matchers, actions, and expectations designed for interacting with content inside web views.
 
-## Locating Web View Elements
+## Locating Elements in Web Views
 
-### Single Web View Scenario
+### `web.element(matcher)`
 
-When dealing with a single web view on the screen, use the `web.element()` function with web view matchers to locate elements within it. Once an element is located, you can apply web view actions and expectations to it.
+In cases where there's only one web view present on the screen, you may use the `web.element()` function, paired with [web view matchers], to reference elements within the web view.
+Upon obtaining the element reference, you can utilize web view actions and expectations on the webView element.
 
-```javascript
+```js
 const innerElement = web.element(by.web.id('inner_element_identifier'));
 await expect(innerElement).toHaveText('Hello World!');
 ```
 
-This example demonstrates locating an element by its HTML `id` attribute and verifying its text content.
+The operation above explains how to locate an inner element using its HTML id attribute and how to verify its text content.
 
-### Multiple Web Views Scenario
+### `web(nativeMatcher).element(matcher)`
 
-For screens with multiple web views, first identify the specific web view using a **[native matcher]**. Then, locate the element within that web view.
+For screens with several web views, it's first necessary to identify a specific web view using a native matcher.
```

Comment:
If you have multiple web views on the screen, you must locate a specific web view first by using a [native matcher][native matchers], e.g.:

```js
const myWebView = web(by.id('webview_identifier'));
```

Following that, you can locate the element within the identified web view:

```js
const innerElement = myWebView.element(by.web.id('inner_element_identifier'));
await expect(innerElement).toHaveText('Hello World!');
```